Header header
uint32[] block_is_on
string gripper_state
int32 grasped_block